src/lib/core/components/visualizations/implementations/MapVisualization.tsx

```diff
@@ -0,0 +1,242 @@
+import { VisualizationProps, VisualizationType } from '../VisualizationTypes';
+import map from './selectorIcons/map.svg';
+import * as t from 'io-ts';
+
+// map component related imports
+import MapVEuMap from '@veupathdb/components/lib/map/MapVEuMap';
+import { defaultAnimationDuration } from '@veupathdb/components/lib/map/config/map.json';
+import geohashAnimation from '@veupathdb/components/lib/map/animation_functions/geohash';
+import { BoundsViewport } from '@veupathdb/components/lib/map/Types';
+import DonutMarker from '@veupathdb/components/lib/map/DonutMarker';
+import { BoundsDriftMarkerProps } from '@veupathdb/components/lib/map/BoundsDriftMarker';
+
+// viz-related imports
+import { PlotLayout } from '../../layouts/PlotLayout';
+import { useDataClient, useStudyMetadata } from '../../../hooks/workspace';
+import { useMemo, useCallback, useState, ReactElement } from 'react';
+import { preorder } from '@veupathdb/wdk-client/lib/Utils/TreeUtils';
+import DataClient, { MapMarkersRequestParams } from '../../../api/DataClient';
+import { useVizConfig } from '../../../hooks/visualizations';
+import { usePromise } from '../../../hooks/promise';
+import {
+  filtersFromBoundingBox,
+  geohashLevelToVariableId,
+  leafletZoomLevelToGeohashLevel,
+} from '../../../utils/visualization';
+
+export const mapVisualization: VisualizationType = {
+  selectorComponent: SelectorComponent,
+  fullscreenComponent: MapViz,
+  createDefaultConfig: createDefaultConfig,
+};
+
+function SelectorComponent() {
+  return (
+    <img
+      alt="Geographic map"
+      style={{ height: '100%', width: '100%' }}
+      src={map}
+    />
+  );
+}
+
+function createDefaultConfig(): MapConfig {
+  return {
+    mapCenterAndZoom: {
+      latitude: 0,
+      longitude: 0,
+      zoomLevel: 1, // TO DO: check MapVEuMap minZoom hardcoded to 2
+    },
+  };
+}
+
+const defaultAnimation = {
+  method: 'geohash',
+  animationFunction: geohashAnimation,
+  duration: defaultAnimationDuration,
+};
+
+type MapConfig = t.TypeOf<typeof MapConfig>;
+// eslint-disable-next-line @typescript-eslint/no-redeclare
+const MapConfig = t.type({
+  mapCenterAndZoom: t.type({
+    latitude: t.number,
+    longitude: t.number,
+    zoomLevel: t.number,
+  }),
+});
+
+function MapViz(props: VisualizationProps) {
+  const {
+    computation,
+    visualization,
+    updateConfiguration,
+    updateThumbnail,
+    filters,
+    dataElementConstraints,
+    dataElementDependencyOrder,
+    starredVariables,
+    toggleStarredVariable,
+    totalCounts,
+    filteredCounts,
```

Comment:
These cause eslint unused variable warnings, though I assume we will use them soon. `updateThumbail` does too, but I'm gonna use it in my PR, of course

Reply:
Yeah I think we should leave them in for now.  This is just one step along the road.